releases/release-5.3.2.md

```diff
@@ -0,0 +1,165 @@
+---
+title: TiDB 5.3.2 Release Notes
+---
+
+# TiDB 5.3.2 Release Notes
+
+Release Date: June xx, 2022
+
+TiDB version: 5.3.2
+
+## Compatibility Changes
+
++ TiDB
+
+    (dup: release-5.2.4.md > Bug fixes> TiDB)- Fix the issue that the `REPLACE` statement incorrectly changes other rows when the auto ID is out of range [#29483](https://github.com/pingcap/tidb/issues/29483)
+
++ PD
+
+    (dup: release-6.1.0.md > Improvements> PD)- Disable compiling swagger server by default [#4932](https://github.com/tikv/pd/issues/4932)
+
+## Improvements
+
++ TiKV
+
+    (dup: release-5.4.0.md > Improvements> TiKV)- Reduce the system call by the Raft client and increase CPU efficiency [#11309](https://github.com/tikv/tikv/issues/11309)
+    (dup: release-6.1.0.md > Improvements> TiKV) - Improve the health check to detect unavailable Raftstore, so that the TiKV client can update Region Cache in time [#12398](https://github.com/tikv/tikv/issues/12398)
```

Comment:
```suggestion
    (dup: release-6.1.0.md > Improvements> TiKV)- Improve the health check to detect unavailable Raftstore, so that the TiKV client can update Region Cache in time [#12398](https://github.com/tikv/tikv/issues/12398)
```